src/components/input-time-picker/input-time-picker.e2e.ts

```diff
@@ -85,7 +94,32 @@ describe("calcite-input-time-picker", () => {
     expect(await input.getProperty("value")).toBe("");
   });
 
-  it("programmatically changing the value reflects in the input for 24-hour (french lang)", async () => {
+  it("opens the time picker on input keyboard focus", async () => {
```

Comment:
This behavior goes against https://github.com/Esri/calcite-components/pull/6834/files#diff-46c8f2a23aecffc29feadb0c27be1a79e2d318ca8859edd29b1f00534da9ff70R380 and will therefore be a regression.

Reply:
oh right, I forgot I saw this PR.  #oldhabitsdiehard

Reply:
Cool, I removed this test and the one right after since the newer "toggle" tests supersede them.